frontend/src/App.js

```diff
@@ -2,12 +2,14 @@ import React from 'react';
 import BlogDetail from './BlogDetail/BlogDetail';
 import { Route, Routes } from 'react-router-dom';
 import HomePage from './HomePage/HomePage';
+import BlogList from './BlogList/BlogList';
 class App extends React.Component {
 	render() {
 		return (
 			<Routes>
 				<Route path='/' element={<HomePage />} />
-				<Route path='/blogs/:id' element={<BlogDetail />} />
+				<Route path='/posts/:slug' element={<BlogDetail />} />
+				<Route path='/posts' element={<BlogList />} />
 			</Routes>
 		);
 	}
```

Comment:
Separate the imports for created and packages module.
Add a new line above the App component.
Change the class the component to function for consistency.
```javascript
import React from 'react';
import { Route, Routes } from 'react-router-dom';

import BlogDetail from './BlogDetail/BlogDetail';
import HomePage from './HomePage/HomePage';
import BlogList from './BlogList/BlogList';

const App = () => {
	return (
		<Routes>
			<Route path='/' element={<HomePage />} />
			<Route path='/posts/:slug' element={<BlogDetail />} />
			<Route path='/posts' element={<BlogList />} />
		</Routes>
	);
}

export default App;

```

Reply:
Done: Changed to function component